playbook/app/pb_kits/playbook/pb_user/_user.scss

```diff
@@ -38,6 +38,9 @@
 
   &[class*=_horizontal]  {
     align-items: center;
+    .content_wrapper > * {
+      padding: 0 0 0 0;
+    }
```

Comment:
Initially felt like the fix should be making the List kit more specific/limited in how it applies padding to children (currently it's applying it to `[class*=_title]` to any child elements, see screenshot below) but now not so sure.  Seems safe enough to apply the above padding rules since the User kit will likely never want padding on those elements but again feels like the long term fix is actually in the List kit.

Curious to hear what others think.

Here's the styles as applied in the browser for reference:
![list_padding](https://p-a6FbDk.t4.n0.cdn.getcloudapp.com/items/Jru47152/de855640-d92e-4a12-beaf-613d2f430b6d.jpg?v=7b09ab05b82f8d7aeddfc316f8226b70)

I guess scoping the styles in the List kit will be difficult since there's no telling how many/what children will be passed.  Makes me think what Christina's got here makes the most since near term.

Reply:
@bh247484 Good call! It totally agree with you. The weird padding that affected the name in the User kit was supposed to only affect the title of the Nav list title, so I updated the affected class name in the CSS. That way, CSS in User kit does not need be be touched.